packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -218,6 +218,8 @@ export class PivotBase extends BaseComponent<IPivotProps, IPivotState> {
         });
         this._keyToIndexMapping[itemKey] = index;
         this._keyToTabIds[itemKey] = this._getTabId(itemKey, index);
+      } else {
+        warn('The children of a Pivot component must be PivotItems.');
```

Comment:
It _might_ be worth slightly changing this phrasing to say `of type PivotItem` for clarity. Not a blocker just a consideration.

If you do make that change please make the same change in the documentation page. No strong opinion here though.

Reply:
It might also noting that they are required to be of that type to render.

>The children of a Pivot component must be of type PivotItem to be rendered.

Because they any _element_ can be provided, they just won't render unless of type `PivotItem`